.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "CUDA Development Container",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": "."
+    },
+    "runArgs": [
+        "--gpus=all"
+    ],
+    "customizations": {
+        "vscode": {
+            "extensions": [
+            ]
+        }
+    },
+    "remoteUser": "root"
+} 
```

Comment:
please insert a newline at the end of file :)

Reply:
done